README.md

```diff
@@ -168,6 +169,14 @@ fullcontact.person.email('foo@bar.com', null, 'https://mycallbackurl.com', 'webh
 });
 ```
 
+The `Person` API's now accept the `macromeasures` param as a boolean
+
+```js
+fullcontact.person.email('opensource@observe.it', null, null, null, null, null, true, function (err, data) {
```

Comment:
@GitTristan since the api would be totally different and therefore cause everything to break, yes it would be a major version change